kernel_tuner/core.py

```diff
@@ -577,9 +595,12 @@ def compile_and_benchmark(self, kernel_source, gpu_args, params, kernel_options,
 
                 # benchmark
                 if func:
+                    # setting the NVML parameters here avoids this time from leaking into the benchmark time, ends up in framework time instead
```

Comment:
```suggestion
                    # Setting the NVML parameters takes a non neglibible amount of time. By setting them 
                    # here, this time is added to the framework time rather than to benchmark time.
```